### PR TITLE
Update vnet-deployment.md

### PR DESCRIPTION
**Pull request by YaelSchuster**
https://github.com/MicrosoftDocs/dataexplorer-docs/issues/1435

### CODE REVIEW

File: data-explorer/vnet-deployment.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The total number of IP addresses:
 > Subnet size must be planned in advance since it can't be changed after Azure Data Explorer is deployed. Therefore, reserve needed subnet size accordingly.
 
 > [!NOTE]
-> Please don't deploy any Azure Resource or Service in the Subnet which is planned to deploy the Azure Data Exploer. Azure Data Explorer also won't resume from suspended if any service or resource running in the subnet.
+> Please don't deploy any Azure Resource or Service in the Subnet which is planned to deploy the Azure Data Explorer. Azure Data Explorer also won't resume from suspended if any service or resource running in the subnet.
```
Comment (YaelSchuster) [inline data-explorer/vnet-deployment.md:58]:
```suggestion
> Please don't deploy any Azure Resource or Service in the Subnet which is planned to deploy Azure Data Explorer. Azure Data Explorer also won't resume from suspended if any service or resource running in the subnet.
```

**Comment by shsagir** [comment]
Learn Build status updates of commit _[f7641b1](https://github.com/YaelSchuster/dataexplorer-docs/commits/f7641b1e2a74f5a7cc67aac6a3a17e7145174277)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/YaelSchuster/dataexplorer-docs/blob/patch-8/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210060936155311-1436/BuildReport?accessString=6f287f1fda4fc3f6490b26df62226fca546aa2ab95b92296263618a03a680992).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by shsagir** [comment]
Learn Build status updates of commit _[98c33ba](https://github.com/YaelSchuster/dataexplorer-docs/commits/98c33ba32ecd393cce6b896160d313dc50a03109)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/YaelSchuster/dataexplorer-docs/blob/patch-8/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210060955304627-1436/BuildReport?accessString=ef2b6240b8e926955f09d16ea9ea2e779bc53b789f9c95e3ff6f81d977cc2018).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/vnet-deployment.md [hunk 2]
```diff
@@ -52,10 +52,9 @@ The total number of IP addresses:
 | **Total** | **#engine_instances + 9** |
 
 > [!IMPORTANT]
-> Subnet size must be planned in advance since it can't be changed after Azure Data Explorer is deployed. Therefore, reserve needed subnet size accordingly.
-
-> [!NOTE]
-> Please don't deploy any Azure Resource or Service in the Subnet which is planned to deploy the Azure Data Exploer. Azure Data Explorer also won't resume from suspended if any service or resource running in the subnet.
+>
+> - Make sure that you plan the subnet size before deploying Azure Data Explorer. Once deployed, the subnet size cannot be changed.
+> - Make sure that you don't deploy any Azure resources or services in the Subnet where you plan to deploy Azure Data Explorer. Doing so will prevent Azure Data Explorer starting when resuming from a suspended state.
```
Comment (cosh) [inline data-explorer/vnet-deployment.md:57]:
```suggestion
> - Make sure that you don't deploy any other Azure resources or services in the Subnet where you plan to deploy Azure Data Explorer. Doing so will prevent Azure Data Explorer starting when resuming from a suspended state.
```

**Review by cosh — APPROVED** [review]

**Comment by shsagir** [comment]
Learn Build status updates of commit _[24966d5](https://github.com/YaelSchuster/dataexplorer-docs/commits/24966d53a1d67daa868085211487658607f77c47)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[data-explorer/vnet-deployment.md](https://github.com/YaelSchuster/dataexplorer-docs/blob/patch-8/data-explorer/vnet-deployment.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/44fbb50c-a02d-79e7-9b6a-0040a309a8dd/202210180623139441-1436/BuildReport?accessString=ab30bbbfc23b75a51f58333a4774c66cd2ca1a59551ee612eb22012e434156c5).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/44fbb50c-a02d-79e7-9b6a-0040a309a8dd?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the learn.microsoft.com <a href="https://review.learn.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Learn support channel</a></li></ul>

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**